client/src/components/Filters/FIlterSlider/FilterSlider.js

```diff
@@ -0,0 +1,104 @@
+import React, {Component} from 'react';
+import "./FilterSlider.scss"
+import ReactSlider from 'react-slider'
+import {toast} from 'react-toastify';
+import store from "../../../store";
+import {changeMinPriceFilter, changeMaxPriceFilter} from "../../../action-creators/filter-action-creator";
+
+
+class FilterSlider extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      maxValue: props.maxValue,
+      minValue: props.minValue,
+      minChosenValue: props.minValue,
+      maxChosenValue: props.maxValue,
+    }
+  }
+
+  handleChange = (chosenValues) => {
+    const [minChosenValue, maxChosenValue] = chosenValues;
+    this.setState({minChosenValue, maxChosenValue})
+  };
+
+  handleInputChange = (e) => {
+    const value = Number(e.target.value.replace(/\D/g, ''));
+    if (Number.isInteger(value)) {
+      switch (e.target.name) {
```

Comment:
don't waste component with such structures, move it to helper, and return value to set state